backend/api/v2beta1/error.proto

```diff
@@ -1,4 +1,4 @@
-// Copyright 2022 The Kubeflow Authors
+// Copyright 2018-2022 The Kubeflow Authors
```

Comment:
nit: this is a new file, so doesn't need to add 2018.

Reply:
Technically, these files are derivatives of the the ones in `v1beta1` folder. In particular, `error.proto` is mostly copy-paste. I believe, we should keep the earliest year (2018) and optionally add 2022.